google/longrunning/operations_pb2_grpc.py

```diff
@@ -1,19 +1,3 @@
-# -*- coding: utf-8 -*-
-
-# Copyright 2020 Google LLC
```

Comment:
Is it OK for this license header to be removed?

Reply:
Oh nope that shouldn't have been removed. Re-added.